libftl/ingest.c

```diff
@@ -92,6 +92,111 @@ OS_THREAD_ROUTINE _ingest_get_rtt(void *data) {
     return 0;
 }
 
+ftl_status_t ftl_find_closest_available_ingest(const char* ingestHosts[], int ingestsCount, char* bestIngestHostComputed)

```

Comment:
If you can, do a quick null check and such on the input params.

Reply:
Sorry, forget to do that change. Will do that now.